test/proto/bookstore.proto

```diff
@@ -134,6 +134,11 @@ service Bookstore {
       get: "/bigbook"
     };
   }
+  rpc PostWildcard(EchoBodyRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      post: "/wildcard/{arg=**}"
+    };
```

Comment:
I am not sure if `fully_decode_reserved_expansion` defined [here](https://github.com/googleapis/googleapis/blob/master/google/api/http.proto#L41)  can be specified as `option` in the proto file.  If it could, I like to support it.  Could you help to hook it up?  

The logic will be if UrlUnescape_spec is default,  check such `option` in the either 'api`  or at `method` level.  if it is true, use it.

Reply:
It can't be, `google.api.http` extension has `google.api.HttpRule` type, but `fully_decode_reserved_expansion` is a field of `google.api.Http`.

Reply:
I guess the intention of `Http` message is to define collective list of all rules, e.g. in external yaml specification or similar.

Reply:
I see.  Thanks.